src/api/db/data/20170306084550_remove_duplicate_repositories.rb

```diff
@@ -0,0 +1,50 @@
+class RemoveDuplicateRepositories < ActiveRecord::Migration[5.1]
+  def up
+    old = CONFIG['global_write_through']
+    CONFIG['global_write_through'] = false
+
+    Repository.transaction do
+      sql = <<-SQL
+        SELECT a.*
```

Comment:
Not sure if that's relevant here (performancewise) but seems like a.id would be enough.